python/example/single_cell_bpo_simple.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+import arbor
+import pandas
+import seaborn
+import sys
+
+try:
+    from bluepyopt import ephys
+except ImportError:
+    raise ImportError("Please install bluepyopt to run this example.")
+
+if len(sys.argv) < 2:
+    print("No JSON file passed to the program")
+    sys.exit(0)
+
+cell_json_filename = sys.argv[1]
+cell_json, morpho, labels, decor = ephys.create_acc.read_acc(cell_json_filename)
+
+# Define the regions and locsets in the model.
+defs = {
+    "root": "(root)",  # the start of the soma in this morphology is at the root of the cell.
+    "stim_site": "(location 0 0.5)",  # site for the stimulus, in the middle of branch 0.
+    "det_site": "(location 0 0.8)",
+}  # end of the axon.
+labels.append(arbor.label_dict(defs))
+
+# Set initial membrane potential to -55 mV
+decor.set_property(Vm=-55)
+# Attach stimuli that inject 4 nA current for 1 ms, starting at 3 and 8 ms.
+decor.place('"root"', arbor.iclamp(10, 1, current=5), "iclamp0")
+decor.place('"stim_site"', arbor.iclamp(3, 1, current=0.5), "iclamp1")
+decor.place('"stim_site"', arbor.iclamp(10, 1, current=0.5), "iclamp2")
+decor.place('"stim_site"', arbor.iclamp(8, 1, current=4), "iclamp3")
+# Detect spikes at the soma with a voltage threshold of -10 mV.
+decor.place('"det_site"', arbor.spike_detector(-10), "detector")
+
+# Create the policy used to discretise the cell into CVs.
+# Use a single CV for the soma, and CVs of maximum length 1 μm elsewhere.
+soma_policy = arbor.cv_policy_single('"soma"')
+dflt_policy = arbor.cv_policy_max_extent(1.0)
+policy = dflt_policy | soma_policy
+decor.discretization(policy)
```

Comment:
I don't know if you like that better, but 
```suggestion
decor.discretization('(replace (single "soma") (max-extent 1))')
```
is equivalent. Just a matter of taste.

Similarly you _can_ build the decor through chaining
```
decor = arbor.decor()
 .place()
 .paint()
 ...
)
```
This is not a request, but I wondered if you knew these shorthands.

Reply:
This didn't work, but I've nevertheless changed the part slightly.